course/grades.py

```diff
@@ -295,43 +309,207 @@ def grade_key(entry):
         })
 
 
+class ExportGradeBookForm(StyledForm):
+    def __init__(self, *args, **kwargs):
+        # type: (*Any, **Any) -> None  # noqa
+        super(ExportGradeBookForm, self).__init__(*args, **kwargs)
+
+        self.fields["user_info_fields"] = forms.ChoiceField(
+            choices=(csv_settings.export_csv_fields_options),
+            initial=csv_settings.export_csv_fields_options[0],
+            label=_("User Attributes to Export"),
+        )
+
+        self.fields["exclude_course_staff"] = forms.BooleanField(
+            required=False,
+            initial=True,
+            label=_("Exclude Course Staff"),
+            help_text=_(
+                "If selected, grades of course staff (participations who "
+                "have '%(permission)s' permissions) will be excluded."
+                % {"permission": pperm.assign_grade}),
```

Comment:
Why not base this on the existing `pperm.included_in_grade_statistics`?

Reply:
That's fair.